ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -143,6 +143,18 @@ conformance_test(
     ],
 )
 
+conformance_test(
+    name = "conformance-test-min-1.14",
+    lf_versions = ["1.14"],
+    ports = [6865],
+    server = ":app",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--participant=participant-id=example,port=6865",
+        "--daml-lf-min-version-1.14",
+    ],
+)
```

Comment:
a (non-conformance) test that actually checks that (non-stable) packages  older than 1.14 are rejected would be a nice to have.

Reply:
Hmm, there is already some tests for this at the engine level:

https://github.com/digital-asset/daml/blob/4b55f1ab6a7734886bb6ecc4bec5568119ccf17b/daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala#L2683

But there's room for a test of the new flag `--daml-lf-min-version-1.14`